charts/scalardb/templates/config-map.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scalardb-config-file
```

It would be better to use `scalardb.fullname`.
```suggestion
  name: {{ include "scalardb.fullname" . }}-config-file
```